R/zzz.R

```diff
@@ -40,3 +40,7 @@ hintr_init_traduire <- function() {
 hintr_translator_unregister <- function() {
   traduire::translator_unregister()
 }
+
+t_ <- function(...) {
+  traduire::t_(..., package = "hintr")
+}
```

Comment:
I should have done this in naomi but maybe worth putting a note at the top here that this is needed because when running tests it sometimes picking up the package environment as `testthat` instead of `hintr` as it should.

Reply:
done